lib/datatrans/json/transaction/settle.rb

```diff
@@ -0,0 +1,41 @@
+require 'httparty'
+require 'datatrans/json/transaction/response'
+
+class Datatrans::JSON::Transaction
```

Comment:
should be `module` here, shouldn't it?

however, it is already wrong in the existing classes, even in the old XML stuff, so I would suggest to leave it as it is and make a follow-up PR to fix it in all classes.